From 64bac4e0a2125cad64bd3ae0450aef37a5649113 Mon Sep 17 00:00:00 2001
From: Ran Shalit <rshalit@habana.ai>
Date: Sun, 15 Nov 2020 22:35:02 +0200
Subject: [PATCH] [SW-0] hl: ppboot: comment out memcpy call

Comment out call to memcpy, which is done
as part of ppboot checksum verification for
preboot image.
This is done only temporarily, because of alignment
issue which should be resolved later.

Change-Id: I07146159835f9d53ac74e6b13029e05281a37d7d
Signed-off-by: Ran Shalit <rshalit@habana.ai>
---

diff --git a/ppboot/boot_desc.c b/ppboot/boot_desc.c
index e8550d1..1131480 100644
--- a/ppboot/boot_desc.c
+++ b/ppboot/boot_desc.c
@@ -105,8 +105,13 @@
 		desc->loadaddr = ((u64)img_desc[i].load_high[idx] << 32) |
 					img_desc[i].load_low[idx];
 		desc->desc_ver = img_desc[i].desc_ver;
-		memcpy(desc->chksum, img_desc[i].chksum[idx],
-		       ARRAY_SIZE(desc->chksum));
+
+		/*
+		 * Disable chksum copy temporarily, until alignment issue
+		 * is resolved.
+		 */
+		/* memcpy(desc->chksum, img_desc[i].chksum[idx], */
+		/* ARRAY_SIZE(desc->chksum)); */
 
 		return idx;
 	}
